tests/extensions/python_test_files/multiline_singlelinecomment.pyt

```diff
@@ -0,0 +1,5 @@
+# Docstring
+#.. pii: A long description that
+#  spans multiple
+#  lines
```

Comment:
Does the code need the same indent check as the other code you had written?  What about the following example test:
```
#.. pii: A long description that
#  spans multiple
#  lines
# Some comment that comes after the multiple-line annotation
```

Reply:
Maybe this would still work, but your test is not the same as my proposed test.  I want something like:
```
# Docstring
#.. pii: A long description that
#  spans multiple
#  lines
# A comment that is not indented and not part of the above multi-line annotation
#.. pii_types: id, name
```
I just want the test to ensure that the multi-line comment would end if a comment were found that were not indented, and not just a new toggle annotation.

I'm not sure if annotations were meant to be split up like this, so an alternative would be to reverse the order:
```
# Docstring
#.. pii_types: id, name
#.. pii: A long description that
#  spans multiple
#  lines
# A comment that is not indented and not part of the above multi-line annotation
```
Either way, the test should have the un-indented line immediately following the mutli-line annotation.

Reply:
I understand your point. Please check the updated test.